.ci/scripts/run_metrics.sh

```diff
@@ -20,25 +18,31 @@ PGE=$1          # pge to run in docker
 RUNCONFIG=$2    # runconfig file name (not full path)
 DATA_DIR=$3     # data directory for input and output to the docker run command
 IMAGE_NAME=$4   # name of the docker image to run stats on, ex: opera_pge/dswx_hls:jehofman-dev
+SAMPLE_TIME=$5  # Amount of time between samples (seconds)
 
 # will add the docker tag in a future version
-# container_name="${pge_docker_tag}.${PGE}.${RUNCONFIG}"
+# container_name="${pge_docker_tag}_${PGE}_${RUNCONFIG}"
 
 container_name="$PGE"
 
-metrics_collection_start "$container_name"
+metrics_collection_start "$container_name" "$IMAGE_NAME" "$SAMPLE_TIME"
 
-echo "Running pge '$PGE' (image $IMAGE_NAME) using run config '$RUNCONFIG'"
+echo "Running pge '{$PGE}' (image $IMAGE_NAME) using run config '$RUNCONFIG'"
 echo "Sending 'docker run' command"
 
+#docker run --rm --name "$PGE" -u $UID:$(id -g)\
+#  -v "${DATA_DIR}"/runconfig:/home/conda/runconfig:ro \
+#  -v "${DATA_DIR}"/input_dir:/home/conda/input_dir:ro \
+#  -v "${DATA_DIR}"/output_dir:/home/conda/output_dir \
+#  -i --tty opera/proteus:mid_may_2022 sh -ci "python3 proteus-0.1/bin/dswx_hls.py runconfig/dswx_hls.yaml --log output_dir/test_log.log"
+
 docker run --rm --name "${container_name}" -u $UID:$(id -g)\
-  -v ${DATA_DIR}/runconfig:/home/conda/runconfig:ro \
-  -v ${DATA_DIR}/input_dir:/home/conda/input_dir:ro \
-  -v ${DATA_DIR}/output_dir:/home/conda/output_dir \
-  -i --tty ${IMAGE_NAME} \
-  --file /home/conda/runconfig/${RUNCONFIG}
+  -v "${DATA_DIR}"/runconfig:/home/conda/runconfig:ro \
+  -v "${DATA_DIR}"/input_dir:/home/conda/input_dir:ro \
+  -v "${DATA_DIR}"/output_dir:/home/conda/output_dir \
+  -i --tty opera/proteus:mid_may_2022 sh -ci "python3 proteus-0.1/bin/dswx_hls.py runconfig/dswx_hls.yaml --log output_dir/test_log.log"
```

Comment:
This line looks like it still needs to be parameterized with `${IMAGE_NAME}`, `${RUNCONFIG}`, etc..

It also looks like its still using the calling convention of the SAS-only image (the `sh -ci python3 proteus-0.1/bin/dswx_hls.py...` part), where it should be set up to call our PGE images (`--file ${RUNCONFIG}`)